README.md

```diff
@@ -314,7 +314,7 @@ If you are using PHP 5.3 with an autoloader besides Composer, you must load
 ## Contributing ##
 
 Patches and pull requests are encouraged. All code should follow the
-PSR-2 style guidelines. Please include unit tests whenever possible.
+PSR-2 style guidelines. Please include unit tests whenever possible. You may obtain the test data for the maxmind-db folder from https://github.com/maxmind/MaxMind-DB
```

Comment:
These are actually available as a submodule in this repo. You can retrieve them with `git submodule update --init --recursive` or adding a `--recursive` to your initial clone.